src/App.css

```diff
@@ -23,8 +23,10 @@
   width: 240px;
 }
 
-.sort-menu {
+.sort-menu, .filter-menu {
```

Comment:
because 2 classes share the same attributes, it would better to create a more generic class name like `.btn-group`

Reply:
indeed, thanks. not sure if that's the right place to add the filter yet. lets keep this kind mind if we improve this further. 

i think a cool addition would be to make it easy to submit a new issue. i mean it can evolve into something similar to producthunt but more like issuehunt so people can discover and vote on the "extra" things we want to be fixed